dir.props

```diff
@@ -388,7 +393,7 @@
     <MicrosoftNetCoreRuntimeAppxOutputPath>$(UWPOutputDir)Microsoft.NET.CoreRuntime.$(MajorVersion).$(MinorVersion).appx</MicrosoftNetCoreRuntimeAppxOutputPath>
   </PropertyGroup>
 
-  <PropertyGroup>
+  <PropertyGroup Condition="'$(UsingNETSdkCompiler)' != 'true'">
```

Comment:
Merge these two property groups under the same condition?

Reply:
Done.